ios/FluentUI/Bottom Sheet/BottomSheetController.swift

```diff
@@ -91,6 +91,18 @@ public class BottomSheetController: UIViewController {
         }
     }
 
+    /// Preferred height of `expandedContentView`.
+    ///
+    /// The default value is 0, which results in a full screen sheet expansion.
+    @objc open var preferredExpandedContentHeight: CGFloat = 0 {
+        didSet {
+            if isViewLoaded {
+                updateSheetSizingConstraints()
```

Comment:
either here or in side the function should we check if client is giving the wrong value like bigger than the screen size?

Reply:
The screen height limit is enforced through `bottomSheetView.heightAnchor.constraint(lessThanOrEqualTo: maxSheetHeightLayoutGuide.heightAnchor)`, so giving us an arbitrarily high number here is totally valid.